client/src/assets/styles.css

```diff
@@ -8,14 +8,38 @@
   background-color: var(--background);
 }
 
+#appHeader {
+  font-size: 200%;
+  font-weight: bolder;
+  font-style: italic;
+  color: var(--background);
+  background-color: var(--main);
+  padding: 2.5%;
+}
+
+#appHeaderMargin {
+  height: 50px;
+  background-color: var(--accent);
```

Comment:
I don't think you need a separate div for appHeaderMargin,  i think you can just utilize margin-bottom: 50px?